R/get_groups.R

```diff
@@ -0,0 +1,37 @@
+#' Get list of meetup groups using text-based search
+#'
+#' @param api_key Your api key.
+#' @param text string used to filter groups
+#' @param radius can be either "global" (default) or distance in miles in the
+#' range 0-100.
+#' @return Tibble containing the list of selected groups. The table contains 12
+#' columns: name, urlname, who, members, status, created, organizer, lat, lon,
+#' city, country, timezone.
+#'
+#'@examples
+#' \dontrun{
+#' api_key <- Sys.getenv("rladies_api_key")
+#' groups <- get_groups(api_key = api_key, text = "r-ladies")
+#'}
+#' @export
+get_groups <- function(api_key, text = NULL, radius = "global") {
```

Comment:
The other functions have `api_key` as the last argument, can you swap the order?  You can also default to `NULL`.

Reply:
Should we add `country = NULL` as an input as well? (Or we can skip for now and add later). I was looking though the other [search parameters](https://www.meetup.com/meetup_api/docs/find/groups/), and the only one that looked immediately useful was `country`.  It defaults to "US" when used, but we should default to `NULL` so that `radius = "global"` takes over and is not restricted/overriden in any way.